app/src/main/java/com/cornellappdev/android/eatery/model/MealModel.java

```diff
@@ -0,0 +1,53 @@
+package com.cornellappdev.android.eatery.model;
+
+import android.support.annotation.NonNull;
+import org.threeten.bp.LocalDateTime;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by abdullahislam on 2/18/18.
+ */
+
+public class MealModel extends Interval implements Serializable {
+
+  private MealMenuModel menu;
+  private MealType type;
+
+  MealModel(@NonNull LocalDateTime start,
+      @NonNull LocalDateTime end) {
+    super(start, end);
+  }
+
+  // toString() but better
+  public String stringTo() {
+    StringBuilder info = new StringBuilder(
+        type + " on " + " from: " + getStart() + " to: " + getEnd() + "\n");
```

Comment:
i'd rather start this with a format string inside of the stringbuilder

Reply:
It was using string concatenation before, I just quickly converted it to improve performance/adhere to standard practice.